npm-react-tools/README.md

```diff
@@ -45,6 +46,7 @@ option | values | default
 `stripTypes` | `true`: strips out type annotations | `false`
 `es6module` | `true`: parses the file as an ES6 module | `false`
 `nonStrictEs6module` | `true`: parses the file as an ES6 module, except disables implicit strict-mode (i.e. CommonJS modules et al are allowed) | `false`
+`target` | `"es3"`: ECMAScript 3<br>`"es5"`: ECMAScript 5| `"es5"`
```

Comment:
Markdown files don't usually have <br> elements, do they?  Usually markdown is intended to be both human and machine readable, to the extent possible, right?  cc @zpao